src/rpc/protocol.rs

```diff
@@ -7,6 +7,7 @@ use std::time;
 pub const MAX_FILENAME_LENGTH: usize = 4096;
 pub const MAX_DATA_LENGTH: usize = 65536;
 pub const MAX_METADATA_LENGTH: usize = 4096;
+pub const MAX_COPY_LENGTH: usize = 1024 * 8;
```

Comment:
I think this value should be larger to ensure a steady performance increase.

Reply:
why

Reply:
Just multiply with 4? Alright, the current version is OK.